modules/lazyload/module.php

```diff
@@ -131,9 +131,16 @@ public function __construct() {
 						}
 					});
 					}, { rootMargin: '100px 0px 100px 0px' } );
-					lazyloadBackgrounds.forEach( ( lazyloadBackground ) => {
-						lazyloadBackgroundObserver.observe( lazyloadBackground );
+						lazyloadBackgrounds.forEach( ( lazyloadBackground ) => {
+							lazyloadBackgroundObserver.observe( lazyloadBackground );
 					} );
+				};
+				const events = [
+					'DOMContentLoaded',
+					'elementor/widgets/posts/load-more',
```

Comment:
Consider: renaming the event name to be more generic, if another problem will be found in a different widget, you will have to add a new event or change the name of the current one, because it might not be related ti the posts widget.